bedrock/careers/templates/careers/internships.html

```diff
@@ -20,7 +20,7 @@ <h1 class="mzp-c-hero-title">Make an Impact</h1>
         right spot for you.</p>
```

Comment:
We're going to be no-indexing this file until the redesign in 2023, so we can undo the changes for this file :)